src/__tests__/Footer.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import Footer from '../components/Footer';
+
+describe('Footer Component', () => {
+    it('renders the footer with links', () => {
+        render(<Footer />);
+
+        const aboutLink = screen.getByText(/About/i);
+        const termsLink = screen.getByText(/Terms/i);
+        const privacyPolicyLink = screen.getByText(/Privacy Policy/i);
```

Comment:
Better to check for exact match than case insensitive

Reply:
Fixed